### PR TITLE
Fixed most issues due to dependency updates

### PR DESCRIPTION
**Pull request by anirudhlakhotia**
**Fixed:**

-  Predictions
- Maps
- Design

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Covidash
 
-![covidash](https://socialify.git.ci/anandrajaram21/covidash/image?description=1&font=Raleway&language=1&owner=1&pattern=Circuit%20Board&stargazers=1&theme=Dark)
+![covidash](https://socialify.git.ci/anirudhlakhotia/covidash/image?description=1&font=Raleway&language=1&owner=1&pattern=Circuit%20Board&stargazers=1&theme=Dark)
```
Comment (anandrajaram21) [inline README.md]: ummm, i'm guessing this was a mistake from your fork?

File: src/maps.py [hunk 2]
```diff
@@ -10,12 +10,13 @@
 from itertools import chain
 from math import log
 from math import e
+import app as app
 
 chart_studio.tools.set_credentials_file(
-    username="chartstudiouser", api_key="m9KxT5JPEEukONNW8E50"
+    username="anirudhlakhotia", api_key="tcCFHV4CoCJaqmLfANU6"
```
Comment (anandrajaram21) [inline src/maps.py]: ok so for these api keys, what I did was monstrously stupid. Please try and change to using environment variables, using something like dotenv https://pypi.org/project/python-dotenv/

File: src/maps.py [hunk 3]
```diff
@@ -10,12 +10,13 @@
...
 )
 
-mapbox_access_token = "pk.eyJ1IjoiY2hhcnRzdHVkaW91c2VyIiwiYSI6ImNrZXd3bTBoNTA4bnYyemw4N3l5aDN5azIifQ.7e-KoC1KMXr_EKbkahgAQQ"
+mapbox_access_token = "pk.eyJ1IjoiYW5pcnVkaGxha2hvdGlhIiwiYSI6ImNreThyeDl6NDEyeHUyd3BmaWo3Ynp6MTcifQ.o7XgaewOUt_7BPH6F41R0w"
```
Comment (anandrajaram21) [inline src/maps.py]: Same for this too, use dotenv

**Review by anandrajaram21 — CHANGES_REQUESTED** [review]
Please make those changes. 🙏 thenks,. But also, nice work on finally getting everything to run. I gave up kekw

File: src/maps.py (hunk 2 above)
Comment (anirudhlakhotia) [reply src/maps.py]: Yep, fixed it

File: src/maps.py [hunk 4]
```diff
@@ -7,15 +7,19 @@
...
 from math import log
 from math import e
-
+from dotenv import load_dotenv, find_dotenv
+load_dotenv(find_dotenv())
+API_KEY=os.getenv("API_KEY")
+MAPBOX_ACCESS_TOKEN=os.getenv("MAPBOX_ACCESS_TOKEN")
 chart_studio.tools.set_credentials_file(
-    username="chartstudiouser", api_key="m9KxT5JPEEukONNW8E50"
+    username="anirudhlakhotia", api_key=API_KEY
```
Comment (anandrajaram21) [inline src/maps.py]: make username also in dotenv pls

**Review by anandrajaram21 — APPROVED** [review]
👌 awesome changes 🙏 . Just add the mapbox user to the dotenv too, I'll just approve this, you don't have request for review again lol

**Pull request merged**